teuthology/lock/ops.py

```diff
@@ -11,13 +11,31 @@
 from teuthology.config import config
 from teuthology.contextutil import safe_while
 from teuthology.task import console_log
+import teuthology.task.internal
+from teuthology.misc import canonicalize_hostname
 
 import util
 import keys
 
 log = logging.getLogger(__name__)
 
 
+def update_nodes(nodes, reset_os=False):
+    for node in nodes:
+        remote = teuthology.orchestra.remote.Remote(
+            canonicalize_hostname(node))
+        if reset_os:
+            log.info("Updating [%s]: reset os type and version on server", node)
+            inventory_info = dict()
+            inventory_info['os_type'] = ''
+            inventory_info['os_version'] = ''
+            inventory_info['name'] = remote.hostname
+        else:
+            log.info("Updating [%s]: set os type and version on server", node)
```

Comment:
Please, remove [] around %s, it is not style in of this code base. Also it is better to add actual values of os_type and os_version to this log info.